ui/scss/component/_overlay-media.scss

```diff
@@ -0,0 +1,67 @@
+$inner-width: 320px;
+$inner-height: 240px;
+$inner-margin: 24px;
+
+.overlay-media {
+
+  position: relative;
+  transition: all 0.3s cubic-bezier(.55,0,.1,1);
+  display: block;
+
+
+  & .show-page-media {
+    transition: all 0.3s cubic-bezier(.55,0,.1,1);
+  }
+
+  &.minimized {
+    width: 100% !important;
+    position: fixed;
+    top: 0;
+    left: 0;
+    /*bottom: $inner-height + $inner-margin;*/
+    transform: translateY(calc(100vh - 248px));
+    z-index: 99;
```

Comment:
Right now our z-index is very clean:

- 3 layers (1, 2, 3) in the app.
- 2 layers (9999, 10000) for modals and snackbars, respectively.

I suspect z-index: 3 here would be fine, or at most 4.

Reply:
Ok, I need rewrite styles anyway :+1: